proto/engine/v1/electra.go

```diff
@@ -73,7 +73,7 @@ func unmarshalDeposits(requestListInSSZBytes []byte) ([]*DepositRequest, error)
 
 func unmarshalWithdrawals(requestListInSSZBytes []byte) ([]*WithdrawalRequest, error) {
 	if len(requestListInSSZBytes) < wrSize {
-		return nil, errors.New("invalid withdrawal request length, requests should be at least the size of 1 request")
+		return nil, errors.New("invalid withdrawal requests length, requests should be at least the size of 1 request")
```

Comment:
If you want to nitpick here, then the error should be like this:

```suggestion
		return nil, fmt.Errorf("invalid withdrawal requests length, requests should be at least the size of %d requests", wrSize)
```

Otherwise 1 is a magic number and may not match wrSize

Reply:
done